### PR TITLE
Add Twitter GCI hashtags stream

### PR DESCRIPTION
**Pull request by invisible-defects**
Add Twitter widget, streaming GCI hashtags posts.

### CODE REVIEW

File: static/css/main.css [hunk 1]
```diff
@@ -39,6 +39,10 @@ ul {
   width: 40px;
 }
 
+.twitter  {
```
Comment (jayvdb) [inline static/css/main.css]: Rule is empty. (empty-rules)

*Origin: CSSLintBear, Section: `css`.*

**Comment by invisible-defects** [comment]
I'm not good at web, I need help to place widget it in the correct place.
Would be create, if somebody helps me :)

**Comment by LifetimeAlone** [comment]
It also doesn't look like it should :(

File: static/css/main.css [hunk 2]
```diff
@@ -39,6 +39,12 @@ ul {
   width: 40px;
 }
 
+.twitter {
```
Comment (jayvdb) [inline static/css/main.css]: Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

*Origin: CSSLintBear, Section: `css`.*

File: templates/main.html [hunk 3]
```diff
@@ -21,6 +21,13 @@ <h1>Google Code-in 2017 Current Leaders</h1>
     <i>
       The leading participants for each organization are listed randomly. 
     </i>
+    <div class="twitter">
+      <a class="twitter-timeline"  href="https://twitter.com/hashtag/googlecodein"
+         data-widget-id="938830025809432577">Tweets about GCI
+      </a>
+      <script src="static/js/twitter.js"></script>
```
Comment (blazeu) [inline templates/main.html]: The `<script>` tag can be put on the bottom of the body, after `app.js`

File: templates/main.html [hunk 4]
```diff
@@ -21,6 +21,13 @@ <h1>Google Code-in 2017 Current Leaders</h1>
...
+      </a>
```
Comment (blazeu) [inline templates/main.html]: It's not closing any `<a>` tag :P
Remove it.

**Review by blazeu — CHANGES_REQUESTED** [review]
Maybe put the widget at the bottom (after orgs with no leaders).
Lets test it.

File: static/css/main.css [hunk 5]
```diff
@@ -44,6 +44,10 @@ ul {
   width: 40px;
 }
 
+.twitter  {
+  text-align:center;
+}
```
Comment (blazeu) [inline static/css/main.css]: Lets go with 100% width instead of centering

Try the selector `.twitter > iframe`

**Review by blazeu — CHANGES_REQUESTED** [review]

File: static/css/main.css [hunk 6]
```diff
@@ -44,6 +44,10 @@ ul {
   width: 40px;
 }
 
+.twitter  {
```
Comment (jayvdb) [inline static/css/main.css]: Rule is empty. (empty-rules)

*Origin: CSSLintBear, Section: `css`.*

File: static/css/main.css [hunk 7]
```diff
@@ -44,6 +44,10 @@ ul {
...
+  .twirrer > iframe;
```
Comment (jayvdb) [inline static/css/main.css]: Expected RBRACE at line 48, col 3. (errors)

*Origin: CSSLintBear, Section: `css`.*

File: templates/main.html [hunk 8]
```diff
@@ -75,9 +75,15 @@ <h3>
         {{/noLeader}}
       </div>
     {{/noLeader.length}}
+    <div class="'twitter">
```
Comment (blazeu) [inline templates/main.html]: Remove `'`

**Review by blazeu — CHANGES_REQUESTED** [review]

File: static/css/main.css [hunk 9]
```diff
@@ -44,6 +44,11 @@ ul {
   width: 40px;
 }
 
+.twitter > iframe {
```
Comment (jayvdb) [inline static/css/main.css:69]: Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

*Origin: CSSLintBear, Section: `css`.*

File: static/css/main.css [hunk 10]
```diff
@@ -44,6 +44,11 @@ ul {
...
+  text-align: center;
```
Comment (blazeu) [inline static/css/main.css]: No need

File: static/css/main.css [hunk 11]
```diff
@@ -44,6 +44,11 @@ ul {
...
+  width: 100%;
```
Comment (blazeu) [inline static/css/main.css]: Try `100% !important` 

It's being overridden by the inline styling :/
if you check the devtool (inspect element)

**Review by blazeu — CHANGES_REQUESTED** [review]

**Comment by blazeu** [comment]
Now you need to rebase for the new design :sparkles:

File: static/css/main.css [hunk 12]
```diff
@@ -66,6 +66,10 @@ ul {
   width: 40px;
 }
 
+.twitter > iframe {
+  width: 100% !important;
```
Comment (jayvdb) [inline static/css/main.css]: Use of !important (important)

*Origin: CSSLintBear, Section: `css`.*

File: templates/main.html [hunk 13]
```diff
@@ -88,9 +88,15 @@ <h3>
         {{/noLeader}}
       </div>
     {{/noLeader.length}}
+    <div class="twitter">
+      <a class="twitter-timeline"  href="https://twitter.com/hashtag/googlecodein"
+         data-widget-id="938830025809432577">Tweets about GCI
```
Comment (blazeu) [inline templates/main.html]: I see no "Tweets about GCI" in the result

Comment (blazeu) [reply templates/main.html]: probably just remove it since there's a big #googlecodein

File: static/css/main.css [hunk 14]
```diff
@@ -66,6 +66,14 @@ ul {
   width: 40px;
 }
 
+/*csslint important: false*/
+
+.twitter > iframe {
+  width: 100% !important;
```
Comment (blazeu) [inline static/css/main.css]: Try
```
# Ignore CSSLintBear
```
Here

Comment (invisible-defects) [reply static/css/main.css]: Didn't it work like this?

Comment (invisible-defects) [reply static/css/main.css]: Gitmate shows no mistake now

Comment (blazeu) [reply static/css/main.css]: See my chat on Zulip.

**Review by blazeu — APPROVED** [review]

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/45/commits/9defd0a7bf1a1dc964e661843f1c7fefc52e76cc

**Pull request merged**